ipaserver/install/dnskeysyncinstance.py

```diff
@@ -55,10 +55,10 @@ def __init__(self, fstore=None, logger=logger):
             keytab=paths.IPA_DNSKEYSYNCD_KEYTAB
         )
         self.extra_config = [u'dnssecVersion 1', ]  # DNSSEC enabled
-        self.named_uid = constants.NAMED_USER.uid
-        self.named_gid = constants.NAMED_GROUP.gid
-        self.ods_uid = constants.ODS_USER.uid
-        self.ods_gid = constants.ODS_GROUP.gid
+        self.named_uid = constants.NAMED_USER
+        self.named_gid = constants.NAMED_GROUP
+        self.ods_uid = constants.ODS_USER
+        self.ods_gid = constants.ODS_GROUP
```

Comment:
How about your remove these instance attributes completely and use ``constants.NAMED_USER.uid`` everwhere instead? IIRC the DNS code is the only code that has user/group instance vars.